bucket/jcpicker.json

```diff
@@ -13,7 +13,18 @@
             "hash": "0762c6718020b804a5e7286bdc9a53fc772eb3a5f17e2d1ede2584a964bf8242"
         }
     },
-    "bin": "jcpicker.exe",
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\jcpicker.ini\")) {",
+        "   New-Item \"$persist_dir\\jcpicker.ini\" -Force | Out-Null",
+        "}",
+        "if (!(Test-Path \"$persist_dir\\jcpicker.txt\")) {",
+        "   New-Item \"$persist_dir\\jcpicker.txt\" -Force | Out-Null",
+        "}"
```

Comment:
Idiotic to use force to override the file, which is persisted. Always do the cross compare to check from persist dir and create in dir

```pwsh
'.txt', '.ini' | ForEach-Object {
    New-Item "$dir\$_" | Out-Null
}
```